toolchain/check/testdata/interop/cpp/function/union.carbon

```diff
@@ -148,21 +126,15 @@ union U {
 
 auto foo(U) -> void;
 
-// --- fail_todo_import_definition_single_data_member_value_param_type.carbon
+// --- todo_fail_import_definition_single_data_member_value_param_type.carbon
 
 library "[[@TEST_NAME]]";
 
 import Cpp library "definition_single_data_member_value_param_type.h";
 
 fn F() {
   //@dump-sem-ir-begin
-  // CHECK:STDERR: fail_todo_import_definition_single_data_member_value_param_type.carbon:[[@LINE+7]]:3: error: semantics TODO: `Unsupported: parameter type: union U` [SemanticsTodo]
-  // CHECK:STDERR:   Cpp.foo({});
-  // CHECK:STDERR:   ^~~~~~~
-  // CHECK:STDERR: fail_todo_import_definition_single_data_member_value_param_type.carbon:[[@LINE+4]]:3: note: in `Cpp` name lookup for `foo` [InCppNameLookup]
-  // CHECK:STDERR:   Cpp.foo({});
-  // CHECK:STDERR:   ^~~~~~~
-  // CHECK:STDERR:
+  // TODO: This should probably fail once we import members for classes and unions.
```

Comment:
Note, even if you want unions to work at a high level, maybe you should add a diagnostic for cases that have members? Again, unlike classes, I don't think there are short-term plans to make this work. I would tend to expect a diagnostic to continue existing given this is tested and one was previously present.